package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uswds-docs",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Are we upgrading the version here because we've upgraded the `uswds` dependency requirement, or is it because the USAJOBS case study constitutes a new version of the docs?